docs/HowTo/Troubleshoot/Troubleshooting.md

```diff
@@ -0,0 +1,144 @@
+---
+description: Frequently asked questions and answers for troubleshooting Teku
+---
+
+# Troubleshooting
+
+## P2P port conflicts
+
+If Teku fails to start with a `P2P Port 9000 (TCP/UDP) is already in use. Check for other processes
+using this port.` error, it means that Teku is trying to use a network port that is already in
+use.
+
+For example, Teku and Lighthouse both use port 9000 by default for P2P traffic. You can change
+Teku's default port number with the [`--p2p-port`](../../Reference/CLI/CLI-Syntax.md#p2p-port)
+option.
+
+## Unable to lock a keystore file
+
+If Teku fails to start with an `Unexpected error when trying to lock a keystore file` error, this
+could be because the directory containing the keystores is not writable by Teku.
+
+Teku uses a lockfile mechanism for the keystores to prevent two validator clients using the same
+keystores at the same time, similar to the Lighthouse client.
+
+To resolve this issue, try the one of the following:
+
+* Set the permissions of the directory holding the keystores so that it is writable by Teku.
+* Set [`--validators-keystore-locking-enabled`](../../Reference/CLI/CLI-Syntax.md#validators-keystore-locking-enabled)
+    to `false` to disable the locking functionality.
+
+## Keystore file already in use
+
+If Teku fails to start with a `Keystore file <keystore_file>.lock already in use.` error, this
+could mean the keystore file is already being used by a validator client, or Teku has exited
+unexpectedly and did not removed the lock.
+
+Teku uses a lockfile mechanism for the keystores to prevent two validator clients using the same
+keystores at the same time, similar to the Lighthouse client.
+
+To resolve this issue, try the one of the following:
+
+* Manually remove the lock files that are created alongside your keystore files, with `.lock`
+    appended to the filename. Take care not to delete your keystores.
```

Comment:
Should add a note to double check that no other process is using those keys.  If it is you'll likely wind up getting slashed.

Reply:
Done.